Ginger/GingerCoreCommon/Repository/BusinessFlowLib/BusinessFlow.cs

```diff
@@ -104,6 +119,39 @@ public enum eSource
 
         public List<string> VariablesBeforeExec { get; set; }
 
+        private TimeSpan mDevelopmentTime;
+        [IsSerializedForLocalRepository]
+        public TimeSpan DevelopmentTime
+        {
+            get
+            {
+                StopTimer();
+                return mDevelopmentTime;
+            }
```

Comment:
add setter for this property and use it instead of `mDevelopmentTime`

Reply:
handled